### DISCUSSION TITLE
Capitolo 8 - Dopo il listato 8.9 - Espressione per recupero dell'indice

### DISCUSSION THREAD

**Discussion by lorenzoAWD** [lifeisfoo/react-guida-pratica#17] [Errori e refusi]
Nel capitolo 8, dopo il listato 8.9, nel paragrafo "ATTENZIONE", credo che anziché 
```js
allLists.findIndex( (l) => l.id === todo.id )
``` 
debba esserci 
```js
allLists.findIndex( (l) => l.id === preTodo.lisId )
```

**Comment by lifeisfoo** [comment]
Ciao Lorenzo,
sì, hai ragione il codice corretto nelle righe finali del paragrafo _ATTENZIONE_ a pagina 125 (libro cartaceo) è:

    allLists.findIndex( (l) => l.id === preTodo.listId )

In questo caso specifico, visto che durante il processo di aggiornamento del _todo_ l'id della lista non cambia, potremmo anche usare:

    allLists.findIndex( (l) => l.id === updatedTodo.listId )

Grazie per averlo segnalato e soprattutto complimenti per averlo individuato :)

Buona lettura.

**Reply by lorenzoAWD** [reply]
Grazie a te, anche per l'ulteriore opzione. Complimenti per il libro ;)
